src/services/EdFi.DataManagementService.Api.Tests.E2E/Features/ValidationErrors.feature

```diff
@@ -0,0 +1,60 @@
+Feature: ValidationErrors
+    POST a request that has an invalid payload.
+
+Scenario: Post an empty request object
+ When sending a POST request to "data/ed-fi/schools" with body
+  """
+  """
+ Then the response code is 400
+    And the response body is
+"""
+{"detail":"Data validation failed. See errors for details.","type":"urn:dms:bad-request:data","title":"Data Validation Error","status":400,"correlationId":null,"validationErrors":{},"errors":["A non-empty request body is required."]}
+"""
```

Comment:
How difficult would it be to adjust the `detail`, `type`, and `title` to match the ODS/API? Here's the output from the ODS/API:

```json
{
  "detail": "The request could not be processed. See 'errors' for details.",
  "type": "urn:ed-fi:api:bad-request",
  "title": "Bad Request",
  "status": 400,
  "correlationId": "e2cdf8ee-baa3-44a3-a7fb-c3cca2dfd5b1",
  "errors": [
    "A non-empty request body is required."
  ]
}
```

We can leave `validationErrors` in place or remove it from this response - I have no preference.